lib/game.rb

```diff
@@ -0,0 +1,147 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [[0,1,2],
+                      [3,4,5],
+                      [6,7,8],
+                      [0,3,6],
+                      [1,4,7],
+                      [2,5,8],
+                      [0,4,8],
+                      [6,4,2]]
+
+  def initialize(player_1 = Human.new('X'), player_2 = Human.new('O'), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+    @reset_input
+  end
+
+  def current_player
+    board.turn_count.odd? ? player_2 : player_1
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |win_combination|
+      position = { one: board.cells[win_combination[0]],
+                   two:  board.cells[win_combination[1]],
+                   three:  board.cells[win_combination[2]]}
+ 
+      if position[:one] == 'X' && position[:two] == 'X' && position[:three] == 'X'
+        return win_combination
+      elsif position[:one] == 'O' && position[:two] == 'O' && position[:three] == 'O'
+        return win_combination
+      end
+    end
+    false
+  end
+
+  def draw?
+    board.full? && !won? 
+  end
+
+  def winner
+    if won? && board.cells[won?[0]] == player_1.token
+      player_1.token
+    elsif won? && board.cells[won?[0]] == player_2.token
+      player_2.token
```

Comment:
In both cases, you are just returning whatever `board.cells[won?[0]]` equals. Isn't this equivalent?
```
if won?
  board.cells[won?[0]]
end
```